UnityProject/Assets/Scripts/HealthV2/Living/PolymorphicSystems/ReagentPoolSystem.cs

```diff
@@ -13,15 +14,13 @@ public class ReagentPoolSystem : HealthSystemBase
 		public ReagentMix BloodPool = new ReagentMix();
 		public int StartingBlood = 500;
 
-		[SerializeField]
-		[Required("Need to know our limits for how much blood we have and what not.")]
-		private CirculatoryInfo bloodInfo = null;
-		public Chemistry.Reagent CirculatedReagent => bloodType.CirculatedReagent;
-		public CirculatoryInfo BloodInfo => bloodInfo;
+		public int NormalBlood = 500;
+
+		public BloodType bloodType => bloodReagent as BloodType;
```

Comment:
Rename property 'bloodType' to match pascal case naming rules, consider using 'BloodType'.

Reply:
Ryder doesn't think anything need to change?

Reply:
you don't see the squiggly yellow line?

Reply:
yeah